operators/grpc_operators/python/server/entity_servicer.py

```diff
@@ -0,0 +1,100 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import logging
+import queue
+from typing import AsyncIterable
+
+from holohub.grpc_operators.holoscan_pb2 import EntityRequest, EntityResponse
+from holohub.grpc_operators.holoscan_pb2_grpc import EntityServicer
+
+
+class StreamingStatus:
+    def __init__(self):
+        self.last_received_frame_no = -1
+        self.outgoing_frames = 0
+
+
+class HoloscanEntityServicer(EntityServicer):
+    def __init__(self, new_entity_stream_rpc, entity_stream_rpc_complete):
+        self.new_entity_stream_rpc = new_entity_stream_rpc
+        self.entity_stream_rpc_complete = entity_stream_rpc_complete
+        self.logger = logging.getLogger(__name__)
+
+    async def EntityStream(
+        self, request_iterator: AsyncIterable[EntityRequest], context
+    ) -> AsyncIterable[EntityResponse]:
+        incoming_request_queue = queue.Queue()
+        outgoing_response_queue = queue.Queue()
+        app = self.new_entity_stream_rpc(
+            "EndoscopyToolTracking", incoming_request_queue, outgoing_response_queue
```

Comment:
change to application-neutral name here instead of endoscopy-specific given that this is in the general operators folder and not specific to that app?